app/routes/teststeps.py

```diff
@@ -43,73 +44,85 @@ def create_teststep():
-    except Exception as e:
-        return jsonify(str(e)),400
-    if teststep:
-        if has_access_to_project(teststep.project_id,user.id):
-            teststep.delete()
-        else:
-            return jsonify({"Error" : "You do not have access to this project, kindly connect to project admin to make deletions in the project components"}),401
-    else:
-        return jsonify({"error" : "No such teststep exists"}),404
-    return jsonify({"Success" : "teststep deleted successfully"}),200
-
-@teststeps_blueprint.route('/update',methods=["PUT"])
+    except Exception as err:
+        print(str(err))
+        return jsonify({"error": "something went wrong"}), 400
+
+    if not teststep:
+        return jsonify({"error": "No such teststep exists"}), 404
+
+    if not has_access_to_project(teststep.project_id, user.id):
+        return jsonify({"error": "You do not have access to this project, kindly connect to project admin to make deletions in the project components"}), 401
+
+    teststep.delete()
+    return jsonify({"message": "teststep deleted successfully"}), 200
+
+
+@teststeps_blueprint.route('/update', methods=["PUT"])
 @jwt_required()
 def update_teststep():
     req_data = request.json
     user = get_current_user()
     try:
         teststep = req_data.get('id')
         teststep = TestStepModel.query.get(teststep)
-        if teststep:
-            if has_access_to_project(teststep.project_id,user.id):
-                if req_data.get('name'):
-                    name = req_data.get('name')
-                    teststep.name = name
-                if req_data.get('method'):
-                    method = req_data.get('method')
-                    teststep.method = method
-                if req_data.get('endpoint_id'):
-                    endpoint = req_data.get('endpoint_id')
-                    teststep.endpoint_id = endpoint
-                if req_data.get('header_id'):
-                    header = req_data.get('header_id')
-                    teststep.header_id = header
-                if req_data.get('payload_id'):
-                    payload = req_data.get('payload_id')
-                    teststep.payload_id = payload
-                teststep.update({'modified_by' : user.id})
-            else:
-                return jsonify({"Error" : "You do not have access to this project, kindly connect to project admin to make updates in the project components"}),401
-        else:
-            return jsonify({"Error" : "No such Teststep exists"}),404
+        if not teststep:
+            return jsonify({"error": "No such Teststep exists"}), 404
+
+        if not has_access_to_project(teststep.project_id, user.id):
+            return jsonify({"error": "You do not have access to this project, kindly connect to project admin to make updates in the project components"}), 401
+
+        if req_data.get('name'):
+            name = req_data.get('name')
+            teststep.name = name
```

Comment:
I think instead of writing multiple lines only for condition check and update it, we can directly check and store in one linear code 
e.g.
```suggestion
        teststep.name = req_data.get('name') if req_data.get('name') else teststep.name
```

Reply:
check this we can do it or not

Reply:
Hi Tanmay, I have made the changes as above after testing for files where the update does not require any for-loop or complex if-else operation, however in files where such complexity exists, I have not changed them in order to maintain uniformity.

Reply:
Hi tanmay, ignore the above comment, I have implemented your suggestion.